agent/agent.py

```diff
@@ -106,8 +106,8 @@ def load(self, fp):
 
         else:  # Create new
             logger.info("Create new runner [%s]" % fp)
-            module_path = "hasalTask"
-            object_name = "HasalTask"
+            module_path = "iskakalunan"
+            object_name = "Iskakalunan"
```

Comment:
I can have another fix to solve this issue, since I want to modify agent.py this week..